ims/einstein/iscsi/iet/driver.py

```diff
@@ -123,8 +129,9 @@ def __remove_mapping(self, ceph_img_name, rbd_name):
 
     @log
     def restart_server(self):
-        command = "echo {0} | sudo -S service iscsitarget restart".format(
-            self.password)
+        # command = "echo {0} | sudo -S service iscsitarget restart".format(
+        #    self.password)
+        command = "service iscsitarget restart"
         p = subprocess.Popen(command, shell=True, stderr=subprocess.STDOUT,
```

Comment:
I thought, we had to restart it as root, are you planning to add permissions to BMI user or is there any other change that won't need us to restart it as root.